dev-python/pythondialog/pythondialog-3.5.1.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="A Python module for making simple text/console-mode user interfaces"
+HOMEPAGE="http://pythondialog.sourceforge.net/"
+SRC_URI="mirror://sourceforge/pythondialog//${PV}/python3-${P}.tar.bz2"
+
+LICENSE="LGPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~ia64 ~ppc ~sparc ~x86"
+IUSE="doc examples"
+REQUIRED_USE="examples? ( doc )"
+
+RDEPEND="dev-util/dialog"
+
+python_prepare_all() {
+	sed -e "/^    'sphinx.ext.intersphinx',/d" -i doc/conf.py || die
+	distutils-r1_python_prepare_all
+}
+
+distutils_enable_sphinx doc
+
+python_install_all() {
+	if use examples; then
+		distutils_enable_sphinx examples
```

Comment:
This line causes portage to fail with python3_9 because distutils_enable_sphinx adds doc to IUSE, which does not seem to be allowed at this point.

Reply:
Indeed, I should have tested it.  Sorry about that.  It's not specific to python3.9.